doc/src/modules/physics/vector/advanced.rst

```diff
@@ -198,3 +198,34 @@ so dynamic symbols created before or after will print the same way.
 Also note that ``Vector``'s ``.dt`` method uses the ``._t`` attribute of
 ``dynamicsymbols``, along with a number of other important functions and
 methods. Don't mix and match symbols representing time.
+
+Solving Vector Equations
+========================
+
+To solve equations involving vectors, you cannot directly use the solve
+functions on a vector. Instead, you must convert the vector equation to a
+matrix.
```

Comment:
to a set of scalar equations

the fact that you make it a matrix is just an implementation choice

Reply:
Addressed in cbc63c2aecb6b2567ebcf1fc36a2d72a06c815bc